toc_config.py

```diff
@@ -395,15 +395,31 @@ def parse_chapter(docname, doc, parent, module_meta):
 
     # Create categories.
     category_names = app.config.category_names
+    for key, name in category_names.items():
+        category = app.config.categories.get(key, {"name": name})
+        if name != category.get("name", name):
```

Comment:
`category_names` in `conf.py` may set multilingual names. Isn't this comparison a bit suspicious in that case?

```python
category_names = {
    'chapter': {
        'fi': 'Luku',
        'en': 'Chapter',
    },
    'submit': {
        'fi': 'Pisteytetty',
        'en': 'Graded',
    },
    'feedback': {
        'fi': 'Palaute',
        'en': 'Feedback',
    },
}

```